examples/residual.py

```diff
@@ -0,0 +1,21 @@
+import matplotlib.pyplot as plt
+from sklearn import datasets
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.model_selection import train_test_split
+
+from sklearn_evaluation import plot
+
+data = datasets.make_classification(200, 10, n_informative=5, class_sep=0.65)
+X = data[0]
+y = data[1]
+
+X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.3)
+
+est = RandomForestClassifier()
+est.fit(X_train, y_train)
+
+y_pred = est.predict(X_test)
+y_true = y_test
+
+plot.residual(y_true, y_pred)
+plt.show()
```

Comment:
looks like this example isn't showing much. only a few dots appear in the plot. `make_classification` is to generate sample data for a classifier:

<img width="633" alt="image" src="https://user-images.githubusercontent.com/989250/201981587-f39c0ac9-1e03-4a20-b227-5a05f5e7e73c.png">

try using `make_regression`: https://scikit-learn.org/stable/modules/generated/sklearn.datasets.make_regression.html

Reply:
Ohhh sorry about this :( I changed it in my new version but I don't know why it was not included in the commit. I will fix this